jobs/payment-jobs/tasks/ejv_partner_distribution_task.py

```diff
@@ -193,6 +227,18 @@ def _create_ejv_file_for_partner(cls, batch_type: str):  # pylint:disable=too-ma
 
                     control_total += 1
 
+                partial_refund_number: int = 0
+                ejv_content, control_total, partial_refund_number = cls._process_partial_refunds(
```

Comment:
We're missing some sort of external link to CAS. Similar to the EJV INVOICE LINK

Reply:
https://github.com/bcgov-registries/analytics/blob/16b1ae5116639d0f17fe7d301fbb469ba4bd8789/data_warehouse/sql/views/views.sql#L277

Reply:
There's a sequence number for EJV Invoice links, this will bust the flow of that sequence number unfortunately

Reply:
Probably best to rename EJV INVOICE LINKS to something else and reuse that table

Reply:
I reused the ejv invoice links, the invoice_id need update to invoice_partial_refund_id

Reply:
done, renamed to EJV_LINKS, alter invoice_id to link_id, removed foreign constraint, and added link_type column